src/app_core.py

```diff
@@ -106,6 +108,15 @@ def strdef(name, default):
 else:
     app.config["BITCOIN_EXPLORER"] = "https://blockstream.info/"
 
+if os.getenv("MAIL_SERVER"):
```

Comment:
remove these, we know it is just 'mail' and '578'

Reply:
removing the MAIL_ in the app_core

Reply:
interesting.... looks like this is needed in app_core, when the MAIL_SERVER and MAIL_PORT is deleted, you get the following:
```
Traceback (most recent call last):
  File "/app/src/email_utils.py", line 67, in send_email_postfix
    mail.send(message)
  File "/usr/local/lib/python3.10/site-packages/flask_mail.py", line 491, in send
    with self.connect() as connection:
  File "/usr/local/lib/python3.10/site-packages/flask_mail.py", line 144, in __enter__
    self.host = self.configure_host()
  File "/usr/local/lib/python3.10/site-packages/flask_mail.py", line 158, in configure_host
    host = smtplib.SMTP(self.mail.server, self.mail.port)
  File "/usr/local/lib/python3.10/smtplib.py", line 255, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/local/lib/python3.10/smtplib.py", line 341, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 312, in _get_socket
    return socket.create_connection((host, port), timeout,
  File "/usr/local/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/local/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
```
its like the configuration is needed to be here. without it, i think its being defaulted to the values 'localhost' and '25' which is not accesssible within the be_app container.

Reply:
going to do a reverse of removing the MAIL_SERVER and MAIL_PORT from docker-compose.yml and statically adding the config:
```
app.config["MAIL_SERVER"] = "mail"
appconfig["MAIL_PORT"] = 587
```

and test to see if there's a connection and gets recorded as trying to send by mail container.

looks good:
```
2022-07-07T01:33:40.768868+00:00 INFO    postfix/smtpd[3923]: connect from be_app.beryllium_default[172.18.0.6]
2022-07-07T01:33:40.835537+00:00 INFO    postfix/smtpd[3923]: CBF4D6B40: client=be_app.beryllium_default[172.18.0.6]
2022-07-07T01:33:40.839385+00:00 INFO    postfix/cleanup[4139]: CBF4D6B40: message-id=<165715762076.14.12845072410237043084@2f88f3cfb008>
2022-07-07T01:33:40.843416+00:00 INFO    postfix/qmgr[205]: CBF4D6B40: from=<noreply@beryllium-test.zap.me>, size=10233, nrcpt=1 (queue active)
2022-07-07T01:33:40.843683+00:00 INFO    postfix/smtpd[3923]: disconnect from be_app.beryllium_default[172.18.0.6] ehlo=1 mail=1 rcpt=1 data=1 quit=1 commands=5
```